packages/office-ui-fabric-react/src/components/Slider/examples/Slider.Basic.Example.tsx

```diff
@@ -25,16 +41,17 @@ export class SliderBasicExample extends React.Component<{}, ISliderBasicExampleS
           onChange={(value: number) => console.log(value)}
         />
         <Slider
-          label="Snapping slider example"
+          label="Marks example"
           min={0}
-          max={50}
+          max={100}
           step={10}
           defaultValue={20}
           showValue={true}
           onChange={(value: number) => console.log(value)}
-          snapToStep
```

Comment:
Back to the point that you can not just simply remove a functionality in a component by breaking the API contract.

Reply:
ok readded